src/neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -134,6 +127,7 @@ private bool BlockAccount(ApplicationEngine engine, UInt160 account)
             StorageItem storage = engine.Snapshot.Storages.GetOrAdd(key, () => new StorageItem(new byte[1]));
             List<UInt160> accounts = storage.GetSerializableList<UInt160>();
             if (accounts.Contains(account)) return false;
+            if ((accounts.Count + 1) > engine.Limits.MaxStackSize) throw new ArgumentException("Maximum number of blocked accounts exceeded");
```

Comment:
We should divide the accounts into multiple records for storage.

Reply:
Done